### ISSUE TITLE
Ability to hide courses with staff access from front page

### ISSUE THREAD

**Issue by jonatanschroeder** [PrairieLearn/PrairieLearn#6659]
My current front page is pretty crowded given I have been granted access as staff to several courses. While I may need it again in the future, the fact is that I won't need that access very often. It would be nice to have the ability to choose some courses and hide them by default, with some accordion-like functionality to list these again if I ever need it.

**Comment by firasm** [comment]
Agreed, this would be nice... I also have a bunch of courses from different instructors (though probably not as many as you). 

Accordion is nice... But, I kinda like how Canvas does it - using the "Favourited" courses

Sketch of an idea:

<img width="988" alt="Screenshot 2022-11-28 at 9 07 48 PM" src="https://user-images.githubusercontent.com/2507459/204443873-4a690c05-699f-4cc1-85aa-ac67f14bc38d.png">
